flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+      "x86_64"
```

Comment:
this is disappointing. See https://github.com/flathub/flathub/pull/6925/files#r2347161056 on how to fix it.